pkg/db/client.go

```diff
@@ -14,9 +14,17 @@ import (
 
 // Client knows how to connect and interact with the database
 type Client interface {
+	// Snapshot Functions (writeonly)
 	SnapshotBugzilla(bugzilla.Bugs) error
 	//SnapshotTrello() will be here in the future
-	//QueryBugs() and other read functions
+
+	// Queries (readonly)
+	GetLatest() (time.Time, error)
```

Comment:
split these into separate interfaces so that they're easier to mock:

```go
type ReadClient interface {
  ...
}
type WriteClient interface {
  ...
}
type Client interface {
  ReadClient
  WriteClient
}
```